src/components/map/rent-popup.tsx

```diff
@@ -139,6 +139,14 @@ const RentPopup: React.FC<RentPopupProps> = ({
             {selectedStation && selectedStation.name}
           </h2>
 
+          <p
+            className={classNames(
+              (selectedStation && selectedStation.note && 'station-note') ||
+                'invisible',
```

Comment:
I'd just remove the JSX element instead of rendering it and hiding it with CSS. So rather like:

```jsx
{selectedStation && selectedStation.note && (
  <p>...</p>
)}
```